angular/projects/public-nrpti/src/app/pipes/project-link.pipe.ts

```diff
@@ -22,6 +29,19 @@ export class ProjectLinkPipe implements PipeTransform {
       return `<a href="https://lng.gov.bc.ca/project/2/overview" target="_blank">${str}</a>`;
     }
 
+    if (bcmiMine) {
+      if (this.configService.config.ENVIRONMENT) {
+        switch (this.configService.config.ENVIRONMENT) {
+          case 'prod':
+            return `<a href="https://mines.nrs.gov.bc.ca/p/${bcmiMine._id}/overview" target="_blank">${str}</a>`;
+          case 'test':
+            return `<a href="https://bcmi-f00029-test.apps.silver.devops.gov.bc.ca/p/${bcmiMine._id}/overview" target="_blank">${str}</a>`;
+          default:
+            return `<a href="https://bcmi-f00029-dev.apps.silver.devops.gov.bc.ca/p/${bcmiMine._id}/overview" target="_blank">${str}</a>`;
```

Comment:
I wonder if we should put these links into config.* variables /w default values to avoid hardcoding them.

Reply:
Added configurable app urls to config service https://github.com/bcgov/NRPTI/pull/901/commits/1687872d27c11565977f1bed70e4439fc4f8e4d1